### DISCUSSION TITLE
"There was an error with you request"

### DISCUSSION THREAD

**Discussion by luozijian1223** [hahahumble/speechgpt#24] [Q&A]
在 Setting 里输入了 OpenAI API Key 后，输入一个“Hello!”然后发送，屏幕显示"There was an error with you request"。请问要怎么解决？是还没配置成功的原因吗？
![image](https://user-images.githubusercontent.com/49829141/229521421-52683402-47aa-44c4-b8fd-72825b5ffd07.png)

**Comment by hahahumble** [comment]
检查一下 OpenAI API Key 是否可以正常使用以及网络环境是否支持
